errdef/errors.go

```diff
@@ -28,4 +28,5 @@ var (
 	ErrSizeExceedsLimit   = errors.New("size exceeds limit")
 	ErrUnsupported        = errors.New("unsupported")
 	ErrUnsupportedVersion = errors.New("unsupported version")
+	ErrNotTarFile         = errors.New("non-empty tar file expected")
```

Comment:
nit: The error message can be just "not a tar file" or something like that

Reply:
Changed the error string to `invalid tar file`